src/app/_components/PageSection.tsx

```diff
@@ -17,7 +17,7 @@ export function PageSection({
   return (
     <section>
       <SectionDivider title={kicker} />
-      <Heading className="mb-4" tag="h2" variant="3xl">
+      <Heading className="mb-6" tag="h2" variant="3xl">
```

Comment:
Let's not do this here!

Reply:
We need to comb through the designs once Filipa and Heath are done this week either way!